qdata/ir.py

```diff
@@ -317,6 +335,10 @@ def __init__(self, decl_type, opaque=False, **kwargs):
         self.goplist = []
 
     def declaration_str(self):
+        """The serialized declaration definition. Note that this only includes
```

Comment:
the phrase "serialized declaration definition" is confusing here :thinking:

Reply:
```suggestion
        """The serialized declaration. Note that this only includes
```